src/components/icons/event-icon/event-icon.tsx

```diff
@@ -0,0 +1,419 @@
+import { useEffect, useState } from 'react';
+import { thumbnailStyles } from '../../card/styles';
+
+const PILL_PADDING = 13;
+const RIGHT_MARGIN = 16;
+const ICON_SIZE = 180;
+
+const EventIcon = ({ text }: { text?: string }) => {
```

Comment:
This is something I had discussed with Mike which produces an icon that looks like this (notice the city names):

<img width="841" alt="Screen Shot 2022-11-30 at 5 10 38 PM" src="https://user-images.githubusercontent.com/110849018/210860518-0c0f8ff3-a357-493d-8ca7-2a54faf61650.png">

but I don't believe it's possible to implement now because I'm not getting isolated city strings. I left it in here because I thought we might be able to use it in the future, but I could also remove it.